api/test/logs/logger_provider_test.cc

```diff
@@ -4,6 +4,7 @@
 #include "opentelemetry/nostd/string_view.h"
 
 #include <gtest/gtest.h>
+#include <array>
```

Comment:
We would want to sort things alphabetically unless there is a reason not to do so.

Reply:
I just tried moving it but the CI format test moves it back

Reply:
In this way we can ignore it for now.